spec/controllers/items_controller_spec.rb

```diff
@@ -128,6 +127,46 @@
         end
       end
 
+      context "with an already existing item name" do
```

Comment:
We generally use request specs, not controller specs (which are discouraged in new Rails).